From 78010ce3b444fb9308eaffc4d56b2190b3855cac Mon Sep 17 00:00:00 2001
From: Tao Ren <taoren@fb.com>
Date: Mon, 9 May 2022 23:08:05 -0700
Subject: [PATCH] ARM: dts: aspeed: elbert: Enable spi1 controller

Summary: Enable spi1 controller in elbert dts.

Test Plan:
1) "bitbake elbert-image" succeeded.
2) "spi_pim_ver.sh" prints correct PIM version in elbert.

Reviewers: tfang, hya321, joancaneus, binhuang, goldenbug, #openbmc

Reviewed By: goldenbug

Differential Revision: https://phabricator.intern.facebook.com/D37292795
---
 .../dts/aspeed/aspeed-bmc-facebook-elbert.dts  | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
index d9faa02c0e35..0b6c1b22f2bc 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-elbert.dts
@@ -244,3 +244,21 @@ &emmc {
 	max-frequency = <25000000>;
 	bus-width = <4>;
 };
+
+&spi1 {
+	status = "okay";
+	compatible = "aspeed,ast2600-spi-user";
+
+	/* GPIOV0 controls another mux beyond this Chip select on SCM2 */
+	spidev@0 {
+		compatible = "rohm,dh2228fv";
+		spi-max-frequency = <5000000>;
+		reg = <0>;
+	};
+
+	spidev@1 {
+		compatible = "rohm,dh2228fv";
+		spi-max-frequency = <5000000>;
+		reg = <1>;
+	};
+};
-- 
2.43.0

